Modules/Operations/Entities/OfficeLocation.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Modules\operations\Entities;
+
+use Illuminate\Database\Eloquent\Model;
+use Modules\User\Entities\User;
+
+class OfficeLocation extends Model
+{
+    protected $fillable = ['centre_name', 'centre_head_id', 'capacity', 'current_people_count'];
```

Comment:
When you want all the fields as fillable then you can use $guarded attrbite
```suggestion
    protected $guarded = [];
```